### DISCUSSION TITLE
Moving Compose Sources to Remote Storage

### DISCUSSION THREAD

**Discussion by Phil89a** [louislam/dockge#441] [General]
OK, so I have Dockge in and running - its brilliant 👍

I used the default set-up, so my compose files are stored with the dockge container; however I want to store these on my NAS.

How can I amend Dockge so that it will use an SMB share to the NAS ?

Hope this makes sense

Thanks

**Comment by greirson** [comment]
The quick and easy way is to set /opt/stacks as a mounted share on to your NAS. I do this on a couple of my VMs that aren't heavy read/write.

**Reply by Phil89a** [reply]
Ah; further newbie question - how do I change that ?

Thanks

**Reply by greirson** [reply]
It's very dependent on your specific setup. But if you google for "mount SMB share to Linux" you'll find oodles of guides on ways to set it up. 

I use NFS for a bit easier of a setup if your storage can support it.

**Reply by Phil89a** [reply]
Thanks - I will try NFS

**Comment by Phil89a** [comment]
In the end, I went with SMB and this seems to work....

Not sure yet how to handle the Data for the database.

```version: "3.8"
services:
  dockge:
    image: louislam/dockge:1
    container_name: dockge
    restart: unless-stopped
    ports:
      # Host Port : Container Port
      - 3010:5001
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - ./data:/app/data
      - volStacks:/opt/stacks
networks: {}
volumes:
  volStacks:
    driver: local
    driver_opts:
      type: cifs
      device: ${path}/stacks
      o: username=${uid},password=${pwd},uid=1000,gid=1000
```
